src/Components/CreateImageWizard/utilities/useValidation.tsx

```diff
@@ -152,19 +152,22 @@ export function validateSatelliteToken(
     errors.command = 'Invalid or missing token';
     return errors;
   }
-  const expiresInSeconds = getTokenExpirationTime(match[1]);
-  if (expiresInSeconds === undefined) {
+  const satelliteToken = decodeToken(match[1]);
+  if (satelliteToken === undefined) {
     errors.command = 'Invalid or missing token';
     return errors;
   }
-  const tokenRemainingS = tokenValidityRemaining(expiresInSeconds);
-  if (tokenRemainingS <= 0) {
-    errors.command = `The token is expired. Check out the Satellite documentation to extend the token lifetime.`;
-    return errors;
-  }
-  const expirationString = getExpirationString(tokenRemainingS);
-  if (expirationString !== undefined) {
-    errors.expired = `The token expires in ${expirationString}. Check out the Satellite documentation to extend the token lifetime.`;
+
+  if (satelliteToken.exp) {
+    const tokenRemainingS = tokenValidityRemaining(satelliteToken.exp);
+    if (tokenRemainingS <= 0) {
```

Comment:
We don't notify/warn customers about their token having no expiration time, right? I'm not sure what value is set in that case, but I assume 0?

I'm not sure I understand from a distance how this check is different in terms of blocking tokens that cannot expire without notifying the user about it.

Reply:
I assume we check (and when needed block on) how far the expiration date is only for tokens, that have an expiration date set:
```typescript
  if (satelliteToken.exp) {
    ...
  }
```

Reply:
1) If the token is set with no expiration time, there would be no "exp" in the token, afaik.
So for that situation, we would not add any errors or warnings, just let users go through.
Does that sound okay?

Reply:
Makes perfect sense to me :+1:

Reply:
@avitova We can later add a notice that not setting an expiration time is not considered best practice and secure, but that's fine for now!

Reply:
Good idea, opened an issue: https://github.com/osbuild/image-builder-frontend/issues/3248